GUI/MetaDraw/PsmAnnotationViewModel.cs

```diff
@@ -122,6 +122,23 @@ public PsmAnnotationViewModel()
 
             LineSeries[] allIons = new LineSeries[spectrumMzs.Length];
 
+            // draw the remaining unmatched peaks
```

Comment:
please add the following comment:  //we draw the unmatched peaks after we draw the matched peaks so that the matched peaks are visually sitting on top of the unmatched peaks and are therefore readable by the user and on the printout.